include_core/omrutilbase.h

```diff
@@ -159,6 +159,36 @@ J9CAS8Helper(volatile uint64_t *addr, uint32_t compareLo, uint32_t compareHi, ui
 #endif /* !OMR_ENV_DATA64 && (AIXPPC || LINUXPPC) */
 
 
+#if (defined(__riscv) && defined(LINUX))
```

Comment:
The outermost parentheses here are redundant.

Reply:
Already fixed as suggested above.